contracts/StakeLockContract.sol

```diff
@@ -0,0 +1,109 @@
+pragma solidity >=0.4.22;
+
+contract StakeLockContract {
+    /**
+     * @dev Error messages for require statements
+     */
+    string internal constant ALREADY_LOCKED = 'Tokens already locked';
+    string internal constant NOT_LOCKED = 'No tokens locked';
+    string internal constant AMOUNT_ZERO = 'Amount can not be 0';
+
+    uint256 internal constant LOCK_PERIOD_IN_EPOCHS = 3;
```

Comment:
why 3 here? any comment?

Reply:
Other projects like Eth also lock for something like 3 epoch. It's just a good number to lock. Not too short not too long. We will decide the final number later (added comment)